app/models/notification.rb

```diff
@@ -37,6 +37,7 @@ class Notification < ApplicationRecord
     favourite
     poll
     update
+    relationships_severed
```

Comment:
Perhaps we could come up with a better sounding name for the API value

Reply:
Do you have any suggestion?

Reply:
`severed_relationships`? `broken_relationships`?

Reply:
`severed_relationships` seems consistent with the naming used elsewhere